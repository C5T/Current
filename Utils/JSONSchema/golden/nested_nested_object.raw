{"ObjectOrArray":{"fields":{"a":{"ObjectOrArray":{"fields":{"b":{"ObjectOrArray":{"fields":{"c":{"String":{"values":{"d":1},"instances":1,"nulls":0}}},"instances":1,"nulls":0}}},"instances":1,"nulls":0}}},"instances":1,"nulls":0}}