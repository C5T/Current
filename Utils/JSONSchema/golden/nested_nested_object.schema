// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_a_b_Object) {
  CURRENT_FIELD(c, std::string);
};

CURRENT_STRUCT(Schema_a_Object) {
  CURRENT_FIELD(b, Schema_a_b_Object);
};

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(a, Schema_a_Object);
};

using Schema = Schema_Object;
