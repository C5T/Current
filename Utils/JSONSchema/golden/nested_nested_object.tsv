Field	Type	Set	Unset/Null	Values	Details
Schema	Object	1	0	1 field	a
Schema.a	Object	1	0	1 field	b
Schema.a.b	Object	1	0	1 field	c
Schema.a.b.c	String	1	0	1 distinct value	`d` : 1
